native/packages/react-native-bpk-component-button/package.json

```diff
@@ -15,8 +15,10 @@
   },
   "dependencies": {
     "bpk-tokens": "^26.2.0",
+    "lodash": "^4.17.4",
```

Comment:
For `xor`.

Reply:
I was gonna say we can rely on `lodash.xor`, but then I read [a comment from](https://github.com/lodash/lodash/issues/2385#issuecomment-221890679) the creator of lodash himself.